### PR TITLE
Add Ponyfills

### PR DESCRIPTION
**Pull request by Richienb**
## Description

https://github.com/Richienb/awesome-ponyfills

People can use these modules when looking to support older browsers or node versions without polluting the prototype.

## Reviewed PRs

https://github.com/sindresorhus/awesome/pull/1653
https://github.com/sindresorhus/awesome/pull/1660

cc @sindresorhus

### CODE REVIEW

**Comment by Richienb** [comment]
unicorn

**Comment by maehr** [comment]
awesome-lint returns 👌and my eyes did not catch an error either.

**Comment by Jogai** [comment]
The term is polyfills I believe. Also you have to stick to the pr template.

**Comment by Richienb** [comment]
> The term is polyfills I believe

https://github.com/sindresorhus/ponyfill

> Also you have to stick to the pr template.

I followed the instructions on the template.

**Comment by Jogai** [comment]
Ok, didnt know that. Thanks.

I believe you followed the template, but I think here its required to check all the boxes, not remove everything.

**Comment by Richienb** [comment]
@Jogai I was going to, except there were no boxes to check.

**Comment by Jogai** [comment]
My mistake, there are no boxes. But I still think its common practice, if not a requirement to leave the template in.

File: readme.md [hunk 1]
```diff
@@ -165,6 +165,7 @@
 	- [Observables](https://github.com/sindresorhus/awesome-observables#readme)
 	- [npm scripts](https://github.com/RyanZim/awesome-npm-scripts#readme) - Task runner.
 	- [30 Seconds of Code](https://github.com/30-seconds/30-seconds-of-code#readme) - Code snippets you can understand in 30 seconds.
+	- [Ponyfills](https://github.com/Richienb/awesome-ponyfills) - NPM packages for writing cross-platform code.
```
Comment (sindresorhus) [inline readme.md]: The description should describe what a ponyfill is.

Comment (sindresorhus) [reply readme.md]:
```suggestion
	- [Ponyfills](https://github.com/Richienb/awesome-ponyfills#readme) - NPM packages for writing cross-platform code.
```

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example) - https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md

**Comment by sindresorhus** [comment]
> This pull request has a title in the format Add Name of List

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/pull/1692/commits/d6b251d77ad1604a39d8387e98ff00a97a98b60a this still doesn't describe what a ponyfill is.

**Comment by Richienb** [comment]
@sindresorhus Explaining what a Ponyfill is in one sentence is pretty tedious. I've updated the description again.

**Comment by sindresorhus** [comment]
Your Awesome list readme should also explain what it is. It's not enough to just link to a website.

> Ponyfills for writing cross-platform code.

"Cross-platform" is not correct. This is too specific. Ponyfills (and polyfills) are much more.

**Comment by sindresorhus** [comment]
Bump :)

**Comment by Richienb** [comment]
@sindresorhus Please re-review.

**Comment by sindresorhus** [comment]
The description in your readme should be more verbose. It should describe what polyfills are and then what ponyfills are.

---

https://ponyfoo.com/articles/polyfills-or-ponyfills should also be in a `Article` section in the list.

File: readme.md [hunk 2]
```diff
@@ -165,6 +165,7 @@
 	- [Observables](https://github.com/sindresorhus/awesome-observables#readme)
 	- [npm scripts](https://github.com/RyanZim/awesome-npm-scripts#readme) - Task runner.
 	- [30 Seconds of Code](https://github.com/30-seconds/30-seconds-of-code#readme) - Code snippets you can understand in 30 seconds.
+	- [Ponyfills](https://github.com/Richienb/awesome-ponyfills#readme) - Like polyfills but without patching native APIs.
```
Comment (sindresorhus) [inline readme.md:168]:
```suggestion
	- [Ponyfills](https://github.com/Richienb/awesome-ponyfills#readme) - Like polyfills but without overriding native APIs.
```

**Comment by Richienb** [comment]
Fixed!

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1268915959856685056

**Comment by Richienb** [comment]
👍

**Comment by sindresorhus** [comment]
`Like polyfills but without overriding native APIs.` => `[Ponyfills](https://ponyfill.com) are like polyfills but without overriding native APIs.`

**Pull request merged**